doc/changes/changes_2.6.1.md

```diff
@@ -0,0 +1,30 @@
+# Cloud Storage Extension 2.6.1, released 2022-11-21
+
+Code name: Dependency Upgrade to fix Vulnerabilities
```

Comment:
```suggestion
Code name: Dependency Upgrade to fix Vulnerabilities on top of 2.6.0
```
(disambiguation)

Reply:
Please list CVEs / sonatypes

Guessing from the POM: CVE-2022-37865 and CVE-2022-37866